libs/template/src/lib/render-strategies/strategies/noop.strategy.ts

```diff
@@ -4,13 +4,16 @@ import {
 } from '../../core/render-aware/interfaces';
 
 /**
+ * @description
+ *
  * Noop Strategy
  *
- * This strategy is does nothing. It serves for debugging only
+ * This strategy is does nothing. It serves several performance features
+ * as well as a helpful tool when migrating and for debugging.
```

Comment:
```suggestion
 * The no-operation strategy does nothing.
 * It can be a useful tool for performance improvements as well as debugging
 * The `[viewport-prio]` directive use it to limit renderings to only visible components:
 * see: https://github.com/BioPhoton/rx-angular/blob/master/libs/template/src/lib/experimental/viewport-prio/viewport-prio.directive.ts
```

Reply:
Done